bench-tps/src/bench.rs

```diff
@@ -32,6 +35,8 @@ pub enum BenchTpsError {
     AirdropFailure,
 }
 
+const USE_MOVE: bool = true;
```

Comment:
Can you pass this in as a command-line parameter? We're going to want to do native vs Move TPS.

Reply:
can just recompile right?